odl/trafos/wavelet.py

```diff
@@ -571,6 +872,11 @@ def __init__(self, dom, nscales, wbasis, mode):
 
             'per': periodization -- like periodic-padding but gives the
             smallest possible number of decomposition coefficients.
+        axes : sequence of `int`, optional
+            Dimensions in which to calculate the wavelet transform.
+            The sequence's length has to be equal to dimension of the ``grid``
+            `None` means traditional transform along the axes in ``grid``.
+            If axes is given nscales is discarded.
```

Comment:
To silently discard (that's what this text suggests) is not good. The code should raise an exception (`ValueError`) if both options are given, and the text in "Parameters" should say that "this option cannot be combined with xyz" or similar.